src/pubsub/core/callables.py

```diff
@@ -144,63 +149,73 @@ def __init__(self, func: UserListener, firstArgIdx: int, ignoreArgs: Sequence[st
             self.autoTopicArgName = None.
         """
 
-        (allParams, varParamName, varOptParamName, defaultVals) = getargspec(func)
-        if defaultVals is None:
-            defaultVals = []
-        else:
-            defaultVals = list(defaultVals)
+        args, varParamName, varOptParamName, argsDefaults, kwargs, kwargsDefaults, annotations = inspect.getfullargspec(func)
```

Comment:
I'm surprised this line works because getfullargspec is imported from inspect. You sure this is your latest code?